pkg/webhooks/clusterqueue_webhook.go

```diff
@@ -67,6 +67,18 @@ func (w *ClusterQueueWebhook) Default(ctx context.Context, obj runtime.Object) e
 			ReclaimWithinCohort: kueue.PreemptionPolicyNever,
 		}
 	}
+	if cq.Spec.FlavorFungibility == nil {
+		cq.Spec.FlavorFungibility = &kueue.FlavorFungibility{
+			WhenCanBorrow:  kueue.Borrow,
+			WhenCanPreempt: kueue.TryNextFlavor,
+		}
+	}
+	if cq.Spec.FlavorFungibility.WhenCanBorrow == "" {
+		cq.Spec.FlavorFungibility.WhenCanBorrow = kueue.Borrow
+	}
+	if cq.Spec.FlavorFungibility.WhenCanPreempt == "" {
+		cq.Spec.FlavorFungibility.WhenCanPreempt = kueue.TryNextFlavor
+	}
```

Comment:
I don't think we need these defaulters since the crd defaulters inject these default values into these, right?